### ISSUE TITLE
FvwmButtons - does not stay depressed when changing desktops 1.0.5

### ISSUE THREAD

**Issue by jmcunx** [fvwmorg/fvwm3#726]
Thanks for reporting your bug here!  The following template will help with
giving as much information as possible so that it's easier to diagnose and
fix.

## Upfront Information
The issue occurs when you press a button from FvwmButtons or you switch desktops using the keyboard.  The button mapped to the current (new) Desktop does not stay depressed.  This works on 1.0.4

I paired down my fvwm2rc to what is here and it shows the problem (I also attached it)

https://pastebin.com/HUPhJp6j

* Fvwm3 version fvwm3 1.0.5

* Linux distribution Slackware 15.0

* Platform Linux Intel(R) Core(TM) i7-4810MQ CPU @ 2.80GHz

## Expected Behaviour

When using FvwmButtons to change your desktop, the button that maps to the desktop stays depressed

## Actual Behaviour

When using FvwmButtons to change your desktop, the button for Desktop 0 remains depressed.

## Enabling logging

`fvwm3` has a means of logging what it's doing.  Enabling this when
reproducing the issue might help.  To do this, either change the means fvwm3
is started by adding `-v` as in:

```
fvwm3 -v
```

or, once `fvwm3` has loaded, send `SIGUSR2` as in:

```
pkill -USR2 fvwm3
```

The resulting logfile can be found in `$HOME/.fvwm/fvwm3-output.log`

## Steps to Reproduce

You can use the attached fvwm2rc

* Reduce the problem to the smallest `fvwm` configuration example (where
  possible).  Start with a blank config file (`fvwm3 -f/dev/null`) and go from
  there.

* Does the problem also happen with Fvwm2? No

Include your configuration with this issue.

## Does Fvwm3 crash?
No

  - corefiles are enabled (`ulimit -c`)
  - A corefile might be in `$HOME` or `/tmp/`.
  - If you're using Linux (with `systemd`), check `coredumpctl list`.
    `coredumpctl` may need installing separately.

If you find a corefile, install `gdb` and run:

```
gdb /path/to/fvwm3 /path/to/corefile
```

If you're using `coredumpctl` then use:

```
coredumpctl debug
```

Then from within the `(gdb) ` prompt, issue:

```
bt full
```

... and include the output here.

## Extra Information

* Anything else we should know?

* Feel free to take a screen capture or video and upload to this issue if you
  feel it would help.

* Attach `$HOME/.fvwm/fvwm3-output.log` from the step above.
[fvwm2rc.txt](https://github.com/fvwmorg/fvwm3/files/9838544/fvwm2rc.txt)

**Comment by ThomasAdam** [comment]
Hey @jmcunx 

I think I've just inadvertently fixed this by way of #727 -- please can you try your same config on the `ta/gh-721` branch?  It seems to work on that branch, compared to it not working on `master`, for example.

**Comment by jmcunx** [comment]
Hi Thomas,

Yes, ta/gh-721 corrected the issue.  All seems good to me, I closed this issue.

Thanks
John
